docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -388,6 +388,51 @@ The connector supports pushdown for a number of operations:
 * :ref:`limit-pushdown`
 * :ref:`topn-pushdown`
 
+.. _oracle-predicate-pushdown:
+
+Predicate pushdown support
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector does not support pushdown of any predicates on columns that use
+the ``CLOB``, ``NCLOB``, ``BLOB``, or ``RAW(n)`` Oracle database types, or Trino
+data types that :ref:`map to these Oracle database types <oracle-type-mapping>`.
+
+In the following example, the predicate is not pushed down for either query
+since ``name`` is a column of type ``VARCHAR``, which maps to ``NCLOB`` in
+Oracle:
+
+.. code-block:: sql
```

Comment:
typical trino code style is to use :: to end the paragraph above to avoid the code-block .. but personally I prefer this more explicit setup.. so good by me